.goreleaser.yml

```diff
@@ -1,14 +1,11 @@
-# This is an example .goreleaser.yml file with some sane defaults.
```

Comment:
Little comment to this one:

I am confident that we can remove this line and file from the secrets, because Github is exposing the secret via `${{ secret.GITHUB_TOKEN }}` on default.